packages/component/src/BasicTranscript.js

```diff
@@ -219,20 +238,13 @@ const BasicTranscriptContent = ({ animating, sticky }) => {
   );
 };
 
-BasicTranscriptContent.propTypes = {
-  animating: PropTypes.bool.isRequired,
-  sticky: PropTypes.bool.isRequired
-};
-
 const BasicTranscript = ({ className }) => {
```

Comment:
We should consider consolidating `<BasicTranscript />` and  `<BasicTranscriptContent />` - `<BasicTranscript />` is very trivial.

Reply:
Will be fixed in #2939, when we touch the activity and `<BasicTranscript>` again.